www/js/intro.js

```diff
@@ -102,6 +111,27 @@ angular.module('emission.intro', ['emission.splash.startprefs',
     $state.go('root.main.heatmap');
   };
 
+  $scope.jsonfetching = function(url) {
+    data_dict = {
+      'user': {
+        'uuid': ???,
+        'model': ???,
+        'itinerumVersion': '99c',
+        'os': ???,
+        'osVersion': ???,
+        'createdAt': ???
+      },
+      'surveyName': ???
+    };
+    fetch(url, {
```

Comment:
what is `fetch`? you should be using the `$http` service, no?

Reply:
would this be http.post(url, data_dict) instead?